backend/.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: generic
```

Comment:
typescript and javascript here maybe? IDK about this script but looks like a specificity could help